model/ftn/ww3_grib.ftn

```diff
@@ -620,6 +620,9 @@
 !       (17) Projection center flag
 !       (18) Scanning mode
 !
+!  Projection for PolarStereo grid was changed from 
```

Comment:
It can refer to Octet No Table 3.2
! Projection for PolarStereo grid was changed from KGDS( 1) = 6 to KGDS( 1) = 5 (Earth assumed represented by WGS84 - Octet No 15 Table 3.2)

Reply:
better?

Reply:
awesome